requirements.txt

```diff
@@ -0,0 +1 @@
+PyYAML==6.0
```

Comment:
builder had no external dependencies before and that was really nice

I don't see where this code is even using yaml?

Reply:
This is an artifact back when it had YAML support. Will remove!